pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion.go

```diff
@@ -931,27 +931,39 @@ func (t *Test) runCommandFunc(nodes option.NodeListOption, cmd string) stepFunc
 // ARM64 builds are only available on v22.2.0+.
 func (t *Test) chooseUpgradePath() ([]*clusterupgrade.Version, error) {
 	skipVersions := t.prng.Float64() < t.options.skipVersionProbability
+	numUpgrades := t.numUpgrades()
+	currentVersion := clusterupgrade.CurrentVersion()
+
 	isAvailable := func(v *clusterupgrade.Version) bool {
 		if t.clusterArch() != vm.ArchARM64 {
 			return true
 		}
 
 		return v.AtLeast(minSupportedARM64Version)
 	}
+	isOlderThanMinimumBootstrapVersion := func(v *clusterupgrade.Version) bool {
+		return t.options.minimumBootstrapVersion != nil && v.LessThan(t.options.minimumBootstrapVersion)
+	}
 
-	var numSkips int
+	forceSkipVersion := true
```

Comment:
Nit: somewhat confusing naming... it serves the same purpose as disabling skip-version upgrades, right? Why not simply, `disableSkipVersion`? It's also equivalent to returning a singleton; i.e., crudely, `disableSkipVersion => len(chooseUpgradePath()) <= 1`

Reply:
Not necessarily, it refers to this check forcing at least one skip version upgrade:

```
		// If we haven't performed a skip-version upgrade yet, do it. This logic
		// makes sure that, when skip-version upgrades are enabled, it happens
		// when upgrading to the current release, which is the most important
		// upgrade to be tested on any release branch.
```

Once we get at least one, its not that skip versions are disabled, but that its a 50% chance.

I could change it back to `numSkips`, although we don't really care how many skips happened + it didn't actually keep track of that either.